src/Sylius/Bundle/OrderBundle/Command/RemoveExpiredCartsCommand.php

```diff
@@ -36,6 +36,11 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $expirationTime = $this->getContainer()->getParameter('sylius_order.cart_expiration_period');
+        $output->writeln(
+            sprintf('Command will remove carts that have been idle for <info>%s</info>.', $expirationTime)
```

Comment:
Ok, in fact nope :D This command removes expired cart, that cannot even be paid (orders can be paid). Which means "idle" is a good word for me ;)